hi-ml-histopathology/src/histopathology/configs/classification/BaseMIL.py

```diff
@@ -183,12 +183,12 @@ def get_pooling_layer(self) -> Tuple[nn.Module, int]:
 
     def get_test_plot_options(self) -> Set[PlotOption]:
         if self.num_top_slides > 0:
-            return {PlotOption.HISTOGRAM, PlotOption.TOP_BOTTOM_TILES}
+            return {PlotOption.HISTOGRAM, PlotOption.CONFUSION_MATRIX, PlotOption.TOP_BOTTOM_TILES}
         else:
-            return {PlotOption.HISTOGRAM}
+            return {PlotOption.HISTOGRAM, PlotOption.CONFUSION_MATRIX}
```

Comment:
Minor suggestion: the below would be easier to read and avoid duplication
`def get_test_plot_options(self) -> Set[PlotOption]:
  options = {PlotOption.HISTOGRAM, PlotOption.CONFUSION_MATRIX}
   if self.num_top_slides > 0:
         options.update({PlotOption.TOP_BOTTOM_TILES})
    return options`